lib/ViewModels/TerriaViewer.js

```diff
@@ -741,6 +753,15 @@ function initializeTerrainProvider(terriaViewer) {
         terriaViewer._terrainProvider = new EllipsoidTerrainProvider();
         CreditDisplay._cesiumCredit = undefined;
     }
+
+    // add terria logo unless specified not to
+    if (!terriaViewer.terria.configParameters.hideTerriaLogo){
+      const terriaLogo = require('../../wwwroot/images/terria-watermark-onLight.svg');
+      terriaViewer._defaultTerriaCredit = new Credit('<a href="https://terria.io/" target="_blank" rel="noopener noreferrer" ><img src="' + terriaLogo + '" title="Cesium ion"/></a>', true);
+      CreditDisplay._terriaCredit = terriaViewer._defaultTerriaCredit;
```

Comment:
Why assign this to `CreditDisplay`? Doesn't seem to be used.

Reply:
ah no it's not needed, will remove that